src/main/java/mate/controller/driver/AddDriverController.java

```diff
@@ -25,8 +27,10 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             throws IOException, ServletException {
         String name = req.getParameter("name");
         String licenceNumber = req.getParameter("licence_number");
-        Driver driver = new Driver(name, licenceNumber);
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        Driver driver = new Driver(name, licenceNumber, login, password);
         driverService.create(driver);
-        resp.sendRedirect("/drivers/add");
+        resp.sendRedirect("/login");
```

Comment:
why do you redirect to login page?

Reply:
I'm sorry! My mistake! After adding new driver we should redirect him to login page for that he will has ability enter to our site)